server/shared/database/index.js

```diff
@@ -90,14 +90,14 @@ function defineModel(Model, connection = sequelize) {
   const fields = invoke(Model, 'fields', DataTypes, connection) || {};
   const options = invoke(Model, 'options') || {};
   Object.assign(options, { sequelize: connection });
-  wrapMethods(Model, Promise);
   return Model.init(fields, options);
 }
 
 forEach(models, model => {
   invoke(model, 'associate', models);
   addHooks(model, Hooks, models);
   addScopes(model, models);
+  wrapMethods(model, Promise);
```

Comment:
It is important to wrap `Model` methods after invoking `Model.associate` because relations inject association methods to the model prototype like e.g. `getChildren`.